src/components/ProductList/ProductList.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import "./ProductList.css";
+import { BsStarFill } from "react-icons/bs";
+import { useFilter } from "../../context/FilterContext";
+import { useProduct } from "../../context/ProductContext";
+import {
+  getSortedProducts,
+  getFilteredProducts,
+  priceAfterDiscount,
+  putCommasInPrice,
+} from "../../helpers";
+
+export const ProductList = () => {
+  const {
+    state: { products },
+  } = useProduct();
+  const { state } = useFilter();
+
+  const sortedProducts = getSortedProducts(products, state);
+  const filteredProducts = getFilteredProducts(sortedProducts, state);
+
+  return (
+    <div className="products-grid-container">
+      <div className="grid grid-col-2">
+        {filteredProducts &&
+          filteredProducts.map(product => {
+            const {
+              _id,
+              title,
+              brand,
+              starRating,
+              imageSrc,
+              priceInMrp,
+              discountInPercentage,
+              inStock,
+            } = product;
+            const _priceAfterDiscount = priceAfterDiscount(
```

Comment:
I would suggest to use any one naming convention throughout

Reply:
I used the underscore to differentiate between variable & function while retained the same meaning for better readability.